specification/profiling.md

```diff
@@ -24,8 +24,10 @@ data.
 
 ## Call Stack Sampling
 
-An instrumentation library that has profiling capabilities MUST be able to sample call stacks
-at a fixed interval. This default interval MUST default to 1 second.
+An instrumentation library that has profiling capabilities MUST be able to sample 
+call stacks at a fixed interval. This interval MUST be user-configurable at startup 
+via the `SPLUNK_PROFILER_CALL_STACK_INTERVAL` environment variable. When not explicitly
+configured, the profiler SHALL use default period of 1 second.
```

Comment:
```suggestion
configured, the profiler SHOULD use default period of 1 second.
```

Reply:
I suggest merging https://github.com/signalfx/gdi-specification/pull/157 first and then also this PR could also add `SPLUNK_PROFILER_CALL_STACK_INTERVAL` to the table in `configuration.md`. Then the default value will be in the table.